qa-format

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+# format shell scripts
+shfmt -w -l .
```

Comment:
@devkapilbansal won't this format all files (recursively) in the current directory? What is this needed?

Reply:
This format all shell scripts recursively. Shell formatter treats all executable files with no extensions and `.sh` extension as shell scripts.

That's why init and agent files are formatted separately